examples/event_handling/pipong.py

```diff
@@ -1,7 +1,12 @@
-# A matplotlib based game of Pong illustrating one way to write interactive
-# animation which are easily ported to multiple backends
-# pipong.py was written by Paul Ivanov <http://pirsquared.org>
-
+"""
+====
+Pong
+====
+
+A Matplotlib based game of Pong illustrating one way to write interactive 
+animations which are easily ported to multiple backends pipong.py was written 
+by <a href='http://pirsquared.org'>Paul Ivanov</a>.
```

Comment:
I am somewhat not surprised by the original author of that example…
ping @ivanov :D